src/list/components/item-fields.js

```diff
@@ -9,35 +9,51 @@ import React from "react";
 import CopyToClipboardButton from "../../widgets/copy-to-clipboard-button";
 import FieldText from "../../widgets/field-text";
 import Input from "../../widgets/input";
+import Button from "../../widgets/button";
 import LabelText from "../../widgets/label-text";
 import PasswordInput from "../../widgets/password-input";
 import PasswordText from "../../widgets/password-text";
 
 import styles from "./item-fields.css";
 
 const fieldsPropTypes = PropTypes.shape({
+  isPopup: PropTypes.bool,
   origin: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   username: PropTypes.string.isRequired,
   password: PropTypes.string.isRequired,
 });
 
-export function ItemFields({fields, onCopy}) {
+export function ItemFields({fields, onCopy, isPopup}) {
+  const originEl = isPopup ? (
+      <h4 className={styles.popupOrigin}>{new URL(fields.origin).host}</h4>
+  ) : (
+      <a className={styles.originLink} href={fields.origin}>{fields.origin}</a>
+  );
+
+  const originLabel = isPopup ? null : (
+      <Localized id="item-fields-origin">
+      <LabelText>oRIGIn</LabelText>
+      </Localized>
+  );
+
+  const launchButton = isPopup ? null : (
+      <Button theme="normal" title={"launch"}
+              onClick={() => window.open(fields.origin, "_blank")}>
```

Comment:
Sorry for not catching earlier!  I believe the intent is to open the site in the same window as the Lockbox management tab, if possible.

Also, although it seems a little silly, I think we'll want to use the backend's "open_site" message.  It will greatly help in the near future as we add metrics recording.

Reply:
`window.open` in this context doesn't actually open a new window, it opens a new tab.

Happy to switch to using `open_site` for metrics.

Reply:
> `window.open` in this context doesn't actually open a new window, it opens a new tab.

I (re) realize it's supposed to, but for my testing it's opening a new window every time 😞 .  Maybe it's a difference in contexts (webextension vs. external)?

> Happy to switch to using `open_site` for metrics.

This might also have the happy consequence of opening in a new tab on the same window, just like it does in the doorhanger.